proto/task_service.proto

```diff
@@ -72,8 +72,10 @@ message GetDataResponse {
 }
 
 message GetStreamRequest {
-  uint32 up_fragment_id = 1;
-  uint32 down_fragment_id = 2;
+  uint32 up_actor_id = 1;
+  uint32 down_actor_id = 2;
+  uint32 up_fragment_id = 3;
```

Comment:
So previously we are getting things wrong here? We should use actor_id instead of fragment_id for `GetStream`?

Reply:
yes correct, I did ask Martin before and He say the fragment refer to actor.